enterprise-beans/3.2/_index.md

```diff
@@ -25,4 +30,4 @@ Jakarta Enterprise Beans defines an architecture for the development and deploym
 
 # Compatible Implementations
 
-* Jakarta Enterprise Beans 3.2.5
\ No newline at end of file
+* Jakarta Enterprise Beans 3.2.6
```

Comment:
The Compatible Implementation of Enterprise Beans is GlassFish 5.1.
Also need a link.